libsplinter/src/oauth/store/mod.rs

```diff
@@ -16,4 +16,81 @@
 
 mod memory;
 
+use crate::error::InternalError;
```

Comment:
The module doc comment should be updated

Reply:
updated